tests/integration/navigation/navigation_core.js

```diff
@@ -127,7 +127,7 @@ QUnit.asyncTest( "window.history.back() from external to internal page", functio
 QUnit.asyncTest( "external empty page does not result in any contents", function( assert ) {
 	$.testHelper.pageSequence( [
 		function() {
-			$.mobile.changePage( "blank.html" );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", "blank.html" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?